src/bandwidthRtc.ts

```diff
@@ -255,7 +255,7 @@ class BandwidthRtc {
       let response = (await this.ws.call("createParticipant",
         params
       )) as CreateParticipantResponse;
-      return response.participantId;
+      return [response.participantId, response.deviceToken];
```

Comment:
I might recommend returning the full `CreateParticipantResponse` object to the caller, rather than pulling 2/3 of the fields into an array.